SA/SA.cna

```diff
@@ -1060,3 +1060,38 @@ beacon_command_register(
 "Lists system boot time",
 "Usage: uptime");
 
+
+
+alias enum_filter_driver{
+	local('$args $system')
+
+	$system = "";
+
+	if ((size(@_) < 1) || (size(@_) > 2))
+	{
+		berror($1, beacon_command_detail("enum_filter_driver"));
+		berror($1, "Invalid number of arguments");
+		return;
+	}
+	$system = iff(-istrue $2, $2, $null);
+	blog($1, "Retrieving list of filter drivers");
+	$args = bof_pack($1, "Z", $system);
```

Comment:
This needs to be a lowercase z since multibyte is used instead of unicode funcs.

Reply:
Updated in the most recent commit